src/dashboard/src/components/api/views.py

```diff
@@ -45,6 +45,39 @@
 UUID_REGEX = re.compile(r'^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$', re.IGNORECASE)
 
 
+def _api_endpoint(expected_methods, required_fields=[]):
```

Comment:
Good catch - lists can be problematic if they're being modified instead of reassigned inside a function, though that isn't the case here.  Since we didn't end up using the required fields functionality, I'll remove the stub of it.